addons/mail/static/src/core/web/mail_composer_template_selector.xml

```diff
@@ -6,61 +6,67 @@
                 <i class="fa fa-fw fa-ellipsis-v"/>
             </button>
             <t t-set-slot="content">
-                <div class="px-3 pb-1 text-muted">Insert Template</div>
-                <t t-foreach="state.templates" t-as="template" t-key="template_index">
-                    <DropdownItem onSelected="() => this.onLoadTemplate(template)">
-                        <t t-if="template.display_name" t-out="template.display_name"/>
-                        <span t-else="" class="fst-italic">Untitled</span>
-                    </DropdownItem>
-                </t>
-                <div t-if="state.templates.length === 0" class="fst-italic px-3">
-                    No saved templates
-                </div>
-                <DropdownItem t-if="state.templates.length >= this.limit" onSelected="() => this.onSelectTemplateSearchMoreBtnClick()">
-                    <a href="#">Search More...</a>
-                </DropdownItem>
-                <div class="dropdown-divider"/>
-                <Dropdown>
-                    <div>Save as Template</div>
-                    <t t-set-slot="content">
-                        <div class="px-3 pb-1 text-muted">Overwrite Template</div>
+                <div class="mail-composer-template-dropdown-content">
```

Comment:
Hello @hubvd :wave:,

Do you really need to add a wrapping `div` like that?

What about using the `menuClass` props on the `Dropdown` component instead? With this props, you can add a class on the popover element. You should then be able to use that class in CSS to set a maximal width for the popover element and fix the overflow issue.

Reply:
Indeed, it wasn't really needed. I reused `mail-composer-template-dropdown`. The !important is needed because the popover service adds the mw-100 class